src/CSET/operators/constraints.py

```diff
@@ -65,29 +66,138 @@ def generate_var_constraint(varname: str, **kwargs) -> iris.Constraint:
 
 
 def generate_model_level_constraint(
-    model_level_number: Union[int, str], **kwargs
+    model_level: int | list[int], **kwargs
```

Comment:
Maybe we should just use `levels` for the argument, as that will give uniformity between the operators. We already have the information about what the levels are in the operator name.

Reply:
I had the same thought when writing them, but kept differentiating to keep in line with existing plots. Will open an issue to tidy that up afterwards,